### ISSUE TITLE
Include number of failures

### ISSUE THREAD

**Issue by kelunik** [Insolita/php-async-benchmarks#9]
The different clients show different success rates for me, so I guess it would be helpful to include the number of successful vs. unsuccessful requests. It might also be a good idea to investigate the different error rates.

**Comment by Insolita** [comment]
Yes, it make sense. Different clients have a little bit different timeout logic. For ex. guzzle has separated connection timeout.
And at whole the main reason for lower speed is to follow redirects. But it is necessary for many cases

**Comment by Insolita** [comment]
BTW, Amphp client currently is most greedy for memory usage

**Comment by kelunik** [comment]
@Insolita Indeed. With FFI enabled and `nghttp2` installed, it's much lower, but still high.
